src/datasets/resource_scripts/cellxgene_census_tower.sh

```diff
@@ -1,20 +1,121 @@
+    dataset_url: https://cellxgene.cziscience.com/collections/62ef75e4-cbea-454e-a0ce-998ec40223d3
+    dataset_reference: dominguez2022crosstissue
+    dataset_organism: homo_sapiens
+  - id: cxg_gtex_v9
+    species: homo_sapiens
+    census_version: "2023-07-25"
+    obs_value_filter: "dataset_id == '4ed927e9-c099-49af-b8ce-a2652d069333'"
+    obs_batch: donor_id
+    dataset_name: GTEX v9
+    dataset_summary: Single-nucleus cross-tissue molecular reference maps to decipher disease gene function
+    dataset_description: Understanding the function of genes and their regulation in tissue homeostasis and disease requires knowing the cellular context in which genes are expressed in tissues across the body. Single cell genomics allows the generation of detailed cellular atlases in human tissues, but most efforts are focused on single tissue types. Here, we establish a framework for profiling multiple tissues across the human body at single-cell resolution using single nucleus RNA-Seq (snRNA-seq), and apply it to 8 diverse, archived, frozen tissue types (three donors per tissue). We apply four snRNA-seq methods to each of 25 samples from 16 donors, generating a cross-tissue atlas of 209,126 nuclei profiles, and benchmark them vs. scRNA-seq of comparable fresh tissues. We use a conditional variational autoencoder (cVAE) to integrate an atlas across tissues, donors, and laboratory methods. We highlight shared and tissue-specific features of tissue-resident immune cells, identifying tissue-restricted and non-restricted resident myeloid populations. These include a cross-tissue conserved dichotomy between LYVE1- and HLA class II-expressing macrophages, and the broad presence of LAM-like macrophages across healthy tissues that is also observed in disease. For rare, monogenic muscle diseases, we identify cell types that likely underlie the neuromuscular, metabolic, and immune components of these diseases, and biological processes involved in their pathology. For common complex diseases and traits analyzed by GWAS, we identify the cell types and gene modules that potentially underlie disease mechanisms. The experimental and analytical frameworks we describe will enable the generation of large-scale studies of how cellular and molecular processes vary across individuals and populations.
+    dataset_url: https://cellxgene.cziscience.com/collections/a3ffde6c-7ad2-498a-903c-d58e732f7470
```

Comment:
Is there a way we can get this information automatically? And would the config be the best place for this?

Reply:
It's possible to query dataset metadata from cellxgene, but sometimes I make small changes if some of the metadata that is provided on the cellxgene website is not relevant.

However, I could change that and, indeed as you say, fetch it from cellxgene census automatically.

```python
with cellxgene_census.open_soma() as conn:
  dataset_info = conn["census_info"]["datasets"].read().concat().to_pandas()

dataset_info.keys()
```

    Index(['soma_joinid', 'collection_id', 'collection_name', 'collection_doi',
           'dataset_id', 'dataset_title', 'dataset_h5ad_path',
           'dataset_total_cell_count'],
          dtype='object')